pkgs/applications/misc/razergenie/default.nix

```
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, lib, meson, ninja, pkgconfig, qtbase, qttools
+, wrapQtAppsHook
+, enableExperimental ? false
+, includeMatrixDiscovery ? false
+}:
+
+let
+  version = "0.8.1";
+  pname = "razergenie";
+
+in stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
```

We usually declare pname and version as separate attributes, so this should create the same result:
```suggestion
  inherit pname version;
```